build/Targets/RepoToolset/OptimizationData.targets

```
@@ -36,27 +34,48 @@
     </ItemGroup>
   </Target>
 
-  <!-- The IBCMerge tool is internal only and hence only available during official Microbuild runs.  This target is 
-       specifically not gated on the existance of this file.  During an official build it shoudl always be present and 
-       if it's not then an error needs to be raised.
+  <Target Name="_InitializeAssemblyOptimizationWithTargetAssembly">
+    <ItemGroup>
+      <OptimizeAssembly Include="@(IntermediateAssembly)" OptimizationDataDir="$(_ProjectOptimizationDataDir)" />
+    </ItemGroup>
+  </Target>
+
+  <!--
+    Merges optimization data to assemblies specified in OptimizeAssembly item group.
+
+    Non-incremental. Calling targets need to handle incremental build if necessary.
+    Runs during any CI build. Performs the actual merge only when IBCMerge tool is available. It is expected to be available in an official build.
+  -->
+  <Target Name="_CalculateIbcArgs">
+    <ItemGroup>
+      <OptimizeAssembly Update="@(OptimizeAssembly)">
```

```suggestion
      <OptimizeAssembly>
```

This is the thing where the whole `Update` attribute is confusingly ignored inside a Target. Since you want everything here, you can just drop it.

---

Ah, yes. Thanks for tip.